index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
+
+      <div id="title">
+        Dr. Norman Borlaug
+        <div id="tribute-info">The man who saved a billion lives</div>
+      </div>
```

Comment:
## Required Changes ♻️

Use proper semantic Tags here as well, kindly checked the reference mentioned above. 
Don't you think that you can use the title in the head section?<img src="https://media0.giphy.com/media/MZQkUm97KTI1gI8sUj/giphy.gif"/>